GUI/GUIControl.cpp

```diff
@@ -884,46 +892,17 @@ void GUIControl::update_displayed_raid_dps_value(const double new_dps_value) {
     change += QString::number(((last_raid_sim_result - previous) / previous) * 100, 'f', 1) + "%";
 
     QString dps = QString::number(last_raid_sim_result, 'f', 2);
+
     qDebug() << "Total Raid DPS:" << dps;
+
     simRaidResultUpdated(dps, change, delta > 0);
 }
 
 void GUIControl::calculate_displayed_dps_value() {
     // Note: intended for local testing to build confidence in thread results
-    update_displayed_dps_value(double(current_char->get_statistics()->get_total_personal_damage_dealt())
-                               / (sim_settings->get_combat_iterations_full_sim() * sim_settings->get_combat_length()));
-}
-
-void GUIControl::update_displayed_tps_value(const double new_tps_value) {
-    double previous = last_personal_sim_result_tps;
-    last_personal_sim_result_tps = new_tps_value;
-    double delta = ((last_personal_sim_result - previous) / previous);
-    QString change = delta > 0 ? "+" : "";
-    change += QString::number(((last_personal_sim_result_tps - previous) / previous) * 100, 'f', 1) + "%";
-
-    QString tps = QString::number(last_personal_sim_result_tps, 'f', 2);
-    qDebug() << "Total TPS: " << tps;
```

Comment:
We should keep this output for consistency. ou can move it right back next to the `Total DPS` print.

Reply:
there was a typo in the output. it now prints "Total DPS", "Raid DPS", and "Total TPS", but not "Raid TPS". Is there any use for Raid TPS? seems irrelevant to me